mcc/GetSetAttrs.c

```diff
@@ -418,16 +419,12 @@ IPTR mSet(struct IClass *cl, Object *obj, struct opSet *msg)
               data->ypos = _mtop(obj) - ti_Data%data->fontheight;
               lines = (-smooth/data->fontheight)+2;
 
-              {
-                struct Layer *layer = data->rport->Layer;
-
-                if(layer->DamageList && layer->DamageList->RegionRectangle)
-                  if(MUI_BeginRefresh(muiRenderInfo(obj),0))
-                  {
-                    MUI_Redraw(obj, MADF_DRAWOBJECT);
-                    MUI_EndRefresh(muiRenderInfo(obj), 0);
-                  }
-              }
+              if(layer->DamageList && layer->DamageList->RegionRectangle)
+                if(MUI_BeginRefresh(muiRenderInfo(obj),0))
```

Comment:
Wouldn't that break the code!?
In this state, MUI_BeginRefresh() won't be called if the previous if()'s conditions are not met.

If we combine them, it will be called.

Note: You know that, these lines are not my implementations, they were there previously, do you? I did not examine their logic. So I'd really prefer to keep them as they are.